crates/bevy_ui/src/render/mod.rs

```diff
@@ -240,6 +253,13 @@ struct UiVertex {
     pub position: [f32; 3],
     pub uv: [f32; 2],
     pub color: u32,
+    pub uv_min: Vec2,
+    pub uv_max: Vec2,
+    pub size: Vec2,
+    pub border_color: u32,
+    pub border_width: f32,
+    /// Radius for each corner in this order: top-left, bottom-left, top-right, bottom-right
+    pub border_radius: [f32; 4],
```

Comment:
Since vertices represent a corner in a given quad at all times you could use a single f32 here saving on bandwidth to the GPU.

Reply:
The code has been changed to use a uniform buffer now so it's no longer necessary to do this.